adminpages/membershiplevels.php

```diff
@@ -1308,186 +1327,225 @@ function update_level_group_order(event, ui) {
+												<?php } ?>
+											</td>
+											<td>
+												<?php if(!pmpro_isLevelExpiring($level)) { ?>
+													--
+												<?php } else { ?>
+													<?php _e('After', 'paid-memberships-pro' );?> <?php echo $level->expiration_number?> <?php echo sornot($level->expiration_period,$level->expiration_number)?>
+												<?php } ?>
+											</td>
+											<td><?php
+												if($level->allow_signups) {
+													if ( ! empty( $pmpro_pages['checkout'] ) ) {
+														?><a target="_blank" href="<?php echo esc_url( add_query_arg( 'level', $level->id, pmpro_url("checkout") ) );?>"><?php esc_html_e('Yes', 'paid-memberships-pro' );?></a><?php
+													} else {
+														_e('Yes', 'paid-memberships-pro' );
+													}
 												} else {
-													_e('Yes', 'paid-memberships-pro' );
+													_e('No', 'paid-memberships-pro' );
 												}
-											} else {
-												_e('No', 'paid-memberships-pro' );
+												?></td>
+											<?php do_action( 'pmpro_membership_levels_table_extra_cols_body', $level ); ?>
+										</tr>
+										<?php
 											}
-											?></td>
-										<?php do_action( 'pmpro_membership_levels_table_extra_cols_body', $level ); ?>
-									</tr>
+										?>
+									</tbody>
+								</table>
+								<p class="text-center">
+									<a class="button button-primary button-hero" href="javascript:addLevel(<?php echo esc_js( $level_group->id ); ?>);">
+										<?php
+											/* translators: a plus sign dashicon */
+											printf( esc_html__( '%s Add New Level', 'paid-memberships-pro' ), '<span class="dashicons dashicons-plus"></span>' ); ?>
+									</a>
+								</p>
+								<div class="pmpro_section_actions">
+									<a class="button-secondary pmpro-has-icon pmpro-has-icon-edit" href="<?php echo esc_url( add_query_arg( array( 'edit_group' => $level_group->id ), admin_url( 'admin.php?page=pmpro-membershiplevels' ) ) ); ?>" ><?php esc_html_e( 'Edit Group', 'paid-memberships-pro' ) ?></a>
 									<?php
-										}
+										// Show a button to delete the group (disabled if there are levels in group).
+										$disabled_button = empty( $group_levels_to_show) ? '' : 'disabled=disabled';
+										$disabled_message = empty( $group_levels_to_show) ? '' : '<span class="description"><em>' . __( 'Move levels to another group to enable group deletion.', 'paid-memberships-pro' ) . '</em></span>';
 									?>
-								</tbody>
-							</table>
-						</div>
+									<button <?php echo esc_attr( $disabled_button ); ?> class="button is-destructive pmpro-has-icon pmpro-has-icon-trash" href="<?php echo esc_url( wp_nonce_url( admin_url( 'admin.php?page=pmpro-membershiplevels&group_id=' . $level_group->id . '&action=delete_group' ), 'delete_group', 'pmpro_membershiplevels_nonce' ) ); ?>" ><?php esc_html_e( 'Delete Group', 'paid-memberships-pro' ) ?></button>
```

Comment:
```suggestion
									<a <?php echo esc_attr( $disabled_button ); ?> class="button is-destructive pmpro-has-icon pmpro-has-icon-trash" href="<?php echo esc_url( wp_nonce_url( admin_url( 'admin.php?page=pmpro-membershiplevels&group_id=' . $level_group->id . '&action=delete_group' ), 'delete_group', 'pmpro_membershiplevels_nonce' ) ); ?>" ><?php esc_html_e( 'Delete Group', 'paid-memberships-pro' ) ?></a>
```
Fixing button not working